multiaccounts/settings/columns.go

```diff
@@ -523,6 +523,11 @@ var (
 		dBColumnName:   "last_tokens_update",
 		valueHandler:   TimeHandler,
 	}
+	ThirdpartyServicesEnabled = SettingField{
```

Comment:
@Khushboo-dev-cpp do we need to sync this field?

Reply:
I understand what you mean, the case of already paired devices , is the one that needs to be discussed further to see what makes sense.


By default true is because else we never give opportunity for user to explore certain features.

Reply:
Perhaps

- Should be synced when `logging in via syncing`
- Not be synced in between `paired devices `

@alaibe @saledjenic 

but I am not sure how to differentiate the two in code lol

Reply:
@saledjenic we have a way for the user to decide privacy mode during onboarding see this design https://www.figma.com/design/idUoxN7OIW2Jpp3PMJ1Rl8/Settings----Desktop-Legacy?node-id=25914-19024&m=dev